src/application/utils/InputValidator.js

```diff
@@ -0,0 +1,9 @@
+import ERROR from '../../constants/Error.js';
+
+export default class InputValidator {
+  static validateInputIsNull (input) {
+    if (input === '' || input === null) {
```

Comment:
input이 null일 경우는 input === '' 보다 먼저 검사하는게 좋지 않을까요?:)
```suggestion
    if (input === '' || input === null) {
```

Reply:
아... 사실 null에 대해 추가할까 말까 고민하다가 넣게 된거라 이런 실수를...